docs/user-event/install.mdx

```diff
@@ -25,12 +25,3 @@ yarn add --dev @testing-library/user-event
 
 </TabItem>
 </Tabs>
-
-Note that `@testing-library/user-event` requires `@testing-library/dom`.
-
-If you use one of the
-[framework wrappers](../dom-testing-library/install.mdx#wrappers), it is
-important that `@testing-library/dom` is resolved to the same installation
-required by the framework wrapper of your choice.  
-Usually this means that if you use one of the framework wrappers, you should not
-add `@testing-library/dom` to your project dependencies.
```

Comment:
I think we should keep this note.
When people upgrade e.g. their version of `@testing-library/react` the `@testing-library/dom` package isn't always deduplicated. I'd rather add examples how to diagnose and fix this to further reduce retyped text when I suspect this to be the root of someeone's problem (e.g. "user-event only types the first N characters").

Reply:
Are you suggesting we revert this deletion and add examples of specific use cases? Could you please give some examples? I'm a bit concerned about the complexity of supporting all this, and now I believe peer dependency errors are in a much better state than they used to be, so I'm not sure it's worth documenting.

Reply:
Yes, I would keep this paragraph on duplicate versions of `/dom`.
I'm under the impression this ranks as the second most common reason for `act` warnings and wrong input with UserEvent after missing `await`. Both isn't our fault and should not happen, but it does and we still have to deal with people running into these problems.
Therefore I'd like to have a section I can point to if I suspect this problem. The exact combinations of packages and package managers change over time and maintaining a list of them is out of scope of our docs. But if we feel that this paragraph isn't perspicuous, maybe we should add some explanation/example.